src/sqlfluff/dialects/dialect_ansi.py

```diff
@@ -2002,15 +2002,22 @@ class ValuesClauseSegment(BaseSegment):
     match_grammar = Sequence(
         OneOf("VALUE", "VALUES"),
         Delimited(
-            Bracketed(
-                Delimited(
-                    Ref("LiteralGrammar"),
-                    Ref("IntervalExpressionSegment"),
-                    Ref("FunctionSegment"),
-                    Ref("BareFunctionSegment"),
-                    "DEFAULT",  # not in `FROM` clause, rule?
-                    ephemeral_name="ValuesClauseElements",
-                )
+            Sequence(
+                # MySQL uses `ROW` in it's value statement.
+                # Currently SQLFluff doesn't differentiate between
+                # Values statement:
+                # https://dev.mysql.com/doc/refman/8.0/en/values.html
+                # and Values() function (used in INSERT statements):
+                # https://dev.mysql.com/doc/refman/8.0/en/miscellaneous-functions.html#function_values
+                # TODO: split these out in future.
+                Ref.keyword("ROW", optional=True),
+                Bracketed(
+                    Delimited(
+                        "DEFAULT",  # not in `FROM` clause, rule?
+                        Ref("ExpressionSegment"),
```

Comment:
Postgres:
![image](https://user-images.githubusercontent.com/80432516/150573339-a579e0d3-569d-4df2-8951-9d546836d080.png)

MySQL:
![image](https://user-images.githubusercontent.com/80432516/150573478-7bd94cd4-cd4c-4b12-9767-08d8dab0ed65.png)

We can just use ExpressionSegment here

Reply:
😎